sympy/interactive/printing.py

```diff
@@ -117,6 +121,8 @@ def _can_print_latex(o):
             return True
```

Comment:
If `NDimArray` is like `Matrix` in that its elements are assumed printable, you should just add it to this list instead of creating a new `elif` clause.